README.md

```diff
@@ -5,171 +5,45 @@
 [![License][license-src]][license-href]
 [![Nuxt][nuxt-src]][nuxt-href]
 
-
-Are you a Nuxt developer? Or are you familiar with Prisma ORM and want to use it easily with Nuxt? Then this module is for you. 
+Are you a Nuxt developer? Or are you familiar with Prisma ORM and want to use it easily with Nuxt? Then this module is for you.
 
 With this module, you can easily integrate Prisma ORM in your Nuxt app.
 
-<!-- - [✨ &nbsp;Release Notes](/CHANGELOG.md) -->
-<!-- - [🏀 Online playground](https://stackblitz.com/github/your-org/my-module?file=playground%2Fapp.vue) -->
-- [📖 &nbsp;Documentation](https://nuxt-prisma-five.vercel.app/)
-
 ## Features
 
 - Seamlessly set up Prisma CLI, Prisma schema, Prisma Migrate, and Prisma Client
 - Easily access Prisma Studio within Nuxt DevTools
 - Auto-imported `usePrismaClient()` composable for your Vue files
 
-## Quick setup
+### Quick setup
```

Comment:
I don't think this is correct. "Quick setup" isn't a child section of "Features". This should probably go back to being on the same level as "Features" and the following sections (You could make them all h3's though if you prefer.)